bin/setup_dev_db_docker.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+sudo docker-compose exec postgres dropdb nhs-virtual-visit-dev -U postgres
+sudo docker-compose exec postgres createdb nhs-virtual-visit-dev -U postgres
+npm run dbmigrate up
+sudo docker-compose exec postgres psql postgresql://postgres:postgres@localhost/nhs-virtual-visit-dev -f /db/seeds.sql
```

Comment:
Is there anyway way we could avoid duplicating the DATABASE_URL used here and in .env ?

What if we shortened the line to only define the database name and the user ?
(e.g. `sudo docker-compose exec postgres psql -U postgres nhs-virtual-visit-dev` )

Or do we think it's fine to keep it as is since people won't be likely to use a DATABASE_URL different to the one in the readme when using docker ?